src/services/asset-discovery-service.ts

```diff
@@ -35,11 +35,10 @@ export default class AssetDiscoveryService extends PercyClientService {
     profile('-> assetDiscoveryService.browser.newPage')
     this.page = await this.browser.newPage()
     await this.page.setRequestInterception(true)
-    await this.page.setJavaScriptEnabled(false)
     profile('-> assetDiscoveryService.browser.newPage')
   }
 
-  async discoverResources(rootResourceUrl: string, domSnapshot: string): Promise<any[]> {
+  async discoverResources(rootResourceUrl: string, domSnapshot: string, enableJavaScript = false): Promise<any[]> {
```

Comment:
I dunno TS, but reading this makes me wonder if `enableJavaScript` is missing a type declaration?

Reply:
@timhaines yeah on the surface that does stand out. TypeScript just infers the type from the default value so you don't have to declare the type here.

Here's what my editor shows me:

<img width="1201" alt="screen shot 2018-12-10 at 2 18 06 pm" src="https://user-images.githubusercontent.com/2478/49755354-92973380-fc86-11e8-91ae-0b9057bb5578.png">